dali/operators/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endif()
 cuda_add_library(${dali_operator_lib} SHARED ${DALI_OPERATOR_SRCS} ${DALI_OPERATOR_PROTO_OBJ})
 set_target_properties(${dali_operator_lib} PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY "${DALI_LIBRARY_OUTPUT_DIR}")
-target_link_libraries(${dali_operator_lib} PUBLIC ${dali_lib} ${dali_kernel_lib} ${dali_core_lib})
+target_link_libraries(${dali_operator_lib} PUBLIC sndfile ${dali_lib} ${dali_kernel_lib} ${dali_core_lib})
```

Comment:
I think `sndfile` should be private - L57

Reply:
Actually, it needs to be a part of link interface, thus it can't be `PRIVATE`. However, it doesn't really need to be linked at this point, so I made the `INTERFACE` type of linking here.